src/vivarium_csu_alzheimers/constants/data_values.py

```diff
@@ -32,3 +33,21 @@
 SCREENING_SCALE_UP_DIFFERENCE = (
     SCREENING_SCALE_UP_GOAL_COVERAGE - PROBABILITY_ATTENDING_SCREENING_START_MEAN
 )
+
+
+# Gamma distribution parameters from Abie's nb
+mean = 3.75  # Fix mean at midpoint of interval [3.5, 4]
```

Comment:
I think you can skip some of these intermediate variables like mean and variance. Having the infor in a comment like you already do should be fine to calculate GAMMA_SHAPE.